ray_on_golem/server/run.py

```diff
@@ -86,8 +111,10 @@ def main():
     args = parse_sys_args()
     prepare_tmp_dir()
 
-    app = create_application()
-    web.run_app(app, port=args.port)
+    app = create_application(args.port, args.self_shutdown)
+    web.run_app(app, port=args.port, print=None)
```

Comment:
hmm, if we're already using the `app`'s setitem/getitem interface to store arbitrary additional properties, we could store the `port` there as well ... and then, there would be just _one_ place where we define the server's port... 

```suggestion
    app = create_application(args.port, args.self_shutdown)
    web.run_app(app, port=app["port"], print=None)
```

plus, (not not, just a thought) ->  if we're allowing the `port` to be chosen, maybe we should also allow override of the `host` argument?

Reply:
Well, good point, added.

But regarding `host` - I've never changed this from `0.0.0.0` to anything else in my whole career. Even more  - every time where host was not `0.0.0.0` it causes some problems. Having host in that way is enough for my taste.